.github/workflows/test.yml

```diff
@@ -100,6 +100,7 @@ jobs:
           MONGODB_MIGRATIONS_CONNECTION_STRING: ${{ secrets.MONGODB_MIGRATIONS_CONNECTION_STRING }}
           GATSBY_REALM_APP_GRAPHQL_URL: ${{ secrets.GATSBY_REALM_APP_GRAPHQL_URL }}
           GATSBY_ROLLBAR_TOKEN: ${{ secrets.GATSBY_ROLLBAR_TOKEN }}
+          INSTRUMENT: 1
```

Comment:
Nit: I would prefer `true` to be both consistent with the other file and more readable.